Lagom: Add two-stage build for Fuzzers to enable fuzzing generated code

ADKaster opened the pull request:
Fixes #9466

The script could of course use some elbow grease but this build gets the basics down.

Merging this will break oss-fuzz until the [companion merge request](https://github.com/google/oss-fuzz/pull/7309) in the oss-fuzz repository is merged.

linusg reviewed (changes requested):
Typos in first commit message: `LibJs` -> `LibJS`, `Fuzzili` -> `Fuzzilli`

trflynn89 commented on Userland/Libraries/LibAudio/Buffer.h:
```diff
@@ -115,6 +113,14 @@ class Buffer : public RefCounted<Buffer> {
+    // Empty Buffer representation, to avoid tiny anonymous buffers in EOF states
+    Buffer()
+        : m_buffer()
+        , m_id(-1)
+        , m_sample_count(0)
```
This seems like the plain types should be initialized at the declaration:
```c++
    const i32 m_id { -1 };
    const int m_sample_count { 0 };
```

Then this ctor can just be:
```c++
Buffer() = default;
```

ADKaster replied:
oh, yep, that's a cleaner solution. Using = default now.

trflynn89 commented:
LGTM otherwise :)

ADKaster commented:
Changes since last push:

- Fixed Tim's comment about the Buffer commit
- Fixed typos in fuzzer commit body (looks like my dictionary needs some help 😅 )
- Added updated docs to the first commit.
